### PR TITLE
Added LiteLink for URL shorteners

### PR DESCRIPTION
**Pull request by naveenalavilli**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x]  The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -879,8 +879,10 @@ API | Description | Auth | HTTPS | CORS |
 | [Bitly](http://dev.bitly.com/get_started.html) | URL shortener and link management | `OAuth` | Yes | Unknown |
 | [CleanURI](https://cleanuri.com/docs) | URL shortener service | `No` | Yes | Yes |
 | [ClickMeter](https://support.clickmeter.com/hc/en-us/categories/201474986) | Monitor, compare and optimize your marketing links | `apiKey` | Yes | Unknown |
+| [LiteLink](https://litelink.ml/) | Simple URL Shortener | `No` | Yes | Yes |
```
Comment (marekdano) [inline README.md:882]: It looks that `https://litelink.ml/` doesn't work anymore. Can you please update it?

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for the contribution @naveenalavilli!

File: README.md (hunk 1 above)
Comment (skryvets) [reply README.md:882]: Worked for me, example: https://www.litelink.ml/MjQ=

Comment (naveenalavilli) [reply README.md:882]: Yup, works for me too. @marekdano, could you pls re-check and discard the pull request if it fails for you again?

**Review by marekdano — APPROVED** [review]

**Pull request merged**